### PR TITLE
docs: Add CITATION.cff Citation File Format file

### PR DESCRIPTION
**Pull request by eduardo-rodrigues**
As per the discussions on the admins mailing list - being fairly inclusive here.

I will wait for a couple of approvals/days to merge, @reikdas, @jonas-eschle, @matthewfeickert, @benkrikler, @Moelf, @jpivarski, @henryiii, @kratsg. Advance thanks.

### CODE REVIEW

**Comment by eduardo-rodrigues** [comment]
@jonas-eschle, I'm unsure as to which email to add and I lack an ORCID. Let me know.

**Comment by eduardo-rodrigues** [comment]
For completeness:
```
cffconvert --validate --infile CITATION.cff
Citation metadata are valid according to schema version 1.2.0.
```

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  - family-names: "Eschle"
+    given-names: "Jonas"
+    affiliation: "University of Zurich"
+  - family-names: "Feickert"
+    given-names: "Matthew"
+    orcid: "https://orcid.org/0000-0003-4124-7862"
+    affiliation: "University of Wisconsin-Madison"
+  - family-names: Krikler
+    given-names: Benjamin
+    affiliation: RemotelyGreen
```
Comment (benkrikler) [inline CITATION.cff:25]: Thanks for this! No need for an affiliation in my case (and it's not a required field in the CITATION.cff schema) but you could add my Orcid and / or github page:
```suggestion
    orcid: "https://orcid.org/0000-0001-9712-0030"
    website: "https://benkrikler.github.io"
```

Comment (eduardo-rodrigues) [reply CITATION.cff]: Perfect, was a best guess. Thanks @benkrikler.

**Review by kratsg — APPROVED** [review]

**Review by matthewfeickert — APPROVED** [review]
LGTM. :+1: Thanks @eduardo-rodrigues! Merge whenever you please / are happy with responses from others. :)

**Comment by eduardo-rodrigues** [comment]
I think we're good to go with 2 explicit approvals here, one approval from @jpivarski in the email thread, a 😄 from @reikdas and an implicit approval by @benkrikler via his update.

Thanks all.

**Pull request merged**